### PR TITLE
curd operation for post

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Hello, i have Added ADD POST, DELETE POST, EDIT POST functionality, in social media app. kindly review it and let me knoe what do think about it !
Files to check:
src/app/slice/addPostModalSlice.js
src/app/slice/postSlice.js
src/pages/Shared/AddPost/AddPost.js
src/pages/Shared/AddPostModal/AddPostModal.js
src/pages/Shared/Feed/Feed.js
src/pages/Shared/FeedCard/FeedCard.js
pr: https://blithe-in.netlify.app/

### CODE REVIEW

File: src/Helpers/Services/actions.jsx [hunk 1]
```diff
@@ -81,3 +83,38 @@ export const getPostByUsername = async (data) => {
...
+};
+
+////actionEditPost
+export const actionEditPost = async (data) => {
+  const response = await mainApiService("actionEditPost", data);
+  if (response.data &&(response.status === 201 || response.status === 200)) {
+    return response;
+    }else{
+      console.log("response.data.errors:", response.data.errors);
+      // showErrorToast(response.data.errors[0])
```
Comment (MayankKumar10) [inline src/Helpers/Services/actions.jsx:118]: Suggestion: remove unnecessary code.

File: src/backend/controllers/PostController.js [hunk 2]
```diff
@@ -1,3 +1,297 @@
+// import { Response } from "miragejs";
```
Comment (MayankKumar10) [inline src/backend/controllers/PostController.js:1]: Suggestion:  remove this file

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions everything else looks good to me you an merge it.

**Review by mikesch803 — APPROVED** [review]
all apis working great, just a suggestion add toast

**Pull request merged**